### PR TITLE
Document IRONdb old-data logging

### PR DESCRIPTION
**Pull request by esproul**
In cases where the submission of "old" data causes issues, this debug
log facility can be turned on to identify the source.

### CODE REVIEW

File: content/irondb/getting-started/configuration.md [hunk 1]
```diff
@@ -104,6 +104,30 @@ epoch-timestamp suffix denoting when they were created:
...
+`disabled="false"` on the `debug/old_data` log:
+```
+<log name="debug/old_data" disabled="false"/>
+```
+
+The threshold for what is considered "old" is controlled by
+`metric_age_threshold`. The value is a [libmtev time
+duration](https://circonus-labs.github.io/libmtev/config/time_durations.html)
+string, and the default is 7 days. Any data submitted with a timestamp that is
+more than this value in the past will be logged.
```
Comment (bdunavant) [inline content/irondb/getting-started/configuration.md:126]:
```suggestion
older than this value in the past will be logged.
```

**Review by bdunavant — APPROVED** [review]
Suggested, but optional changes.

File: content/irondb/getting-started/configuration.md [hunk 2]
```diff
@@ -104,6 +104,30 @@ epoch-timestamp suffix denoting when they were created:
   * Rotated: 1 hour
   * Retained: 1 week
 
+#### Logging old data submission
+
+Sometimes it may be desirable to log data submissions that are older than some
+threshold, in order to identify the source.  Submitting "too old" data can
```
[the remark this answers is not in the document]
Comment (esproul) [reply content/irondb/getting-started/configuration.md]: Fixed.

File: content/irondb/getting-started/configuration.md [hunk 3]
```diff
@@ -104,6 +104,30 @@ epoch-timestamp suffix denoting when they were created:
...
+cause issues with raw shards reappearing, and causing changes to existing
```
[the remark this answers is not in the document]
Comment (esproul) [reply content/irondb/getting-started/configuration.md]: Good suggestion. I also updated the second half to be worded a bit better.

File: content/irondb/getting-started/configuration.md (hunk 1 above)
Comment (esproul) [reply content/irondb/getting-started/configuration.md]: Rewrote it a bit to be less awkward and hopefully clearer.

**Review by bdunavant — APPROVED** [review]
LGTM

**Review by pamaddox — APPROVED** [review]

**Pull request merged**